docs/src/model_setup/output_writers.md

```diff
@@ -158,6 +158,36 @@ NetCDFOutputWriter scheduled on IterationInterval(1):
 └── file size: 17.8 KiB
 ```
 
+`NetCDFOutputWriter` can also be configured for `outputs` that are interpolated or regridded to a different grid than `model.grid`.
+To use this functionality, the `output_grid` must be passed explicitly when constructing `NetCDFOutputWriter` along with the regridded / interpolated `outputs`.
```

Comment:
Should we mention that once `NetCDFWriter` has `output_grid` passed it can _only_ write outputs on that grid. In other words, you can't have outputs that are the original/full grid and the interpolated/coarse grid in the same `NetCDFWriter` file (for now at least). I feel this can be a source of confusion for users in the future.

Reply:
Also we don't pass `output_grid`. We pass the keyword `grid`, correct?

Reply:
> Also we don't pass output_grid. We pass the keyword grid, correct?

We made it a positional argument. But we can change that and make a keyword argument for sure. What do you think?

Also, I agree 100% that making that more explicit is a good idea. The `grid` and outputs _must_ be consistent, otherwise output writing will fail.

Also, you often write `NetCDFWriter`. I like that name because it's a little more concise ("output" is obvious). Should we change this and `JLD2`?

Reply:
I agree that `NetCDFWriter` is better and that we should also change that for JLD.

Reply:
Ok, after this is merged we will get on that.

Reply:
> > Also we don't pass output_grid. We pass the keyword grid, correct?
> 
> We made it a positional argument. But we can change that and make a keyword argument for sure. What do you think?

Ah, true. I have a bit of a preference for keyword arguments, but I don't feel strongly about it.

> Also, I agree 100% that making that more explicit is a good idea. The `grid` and outputs _must_ be consistent, otherwise output writing will fail.
> 
> Also, you often write `NetCDFWriter`. I like that name because it's a little more concise ("output" is obvious). Should we change this and `JLD2`?

Ah, yeah, sorry. I write that for the exact reason you mentioned, and the fact that `NetCDFOutputWriter` is kinda long lol

I often thought abut opening an issue to suggest that change myself, but haven't gotten around to it. So I'm very much in favor of making that change!

Reply:
Hmm ok I agree it should be a kwarg.

Reply:
I noticed that the docs examples are identical to the doc string examples. Ideally these would be different... there's no point in copy pasting here. It's more efficient to use Documenter to insert the docstring.